internal/distro/rhel9/azure.go

```diff
@@ -30,7 +24,7 @@ var (
 		kernelOptions:       defaultAzureKernelOptions,
 		bootable:            true,
 		defaultSize:         4 * common.GibiByte,
-		pipelines:           vhdPipelines(false),
+		image:               liveImage,
```

Comment:
I think this commit also switches the azure image type over to the new style, but the commit message doesn't say so. Just in case you are changing something in the PR anyway.

Reply:
oooh that's weird.  I might try to separate them actually.

Reply:
Separated.